packages/calcite-components/src/components/chip/chip.tsx

```diff
@@ -357,7 +357,7 @@ export class Chip extends LitElement implements InteractiveComponent, LoadableCo
             : undefined;
     return (
       <InteractiveContainer disabled={disabled}>
-        <div
+        <span
```

Comment:
Is this intentional? If so, how does it address the AT issue? Both `div` and `span` are non-semantic, so I’m curious how this influences how our components are interpreted.

Reply:
It is, yes, there aren't a ton of resources out in the wild, but ultimately folks use them interchangeably to provide support to assistive technologies. If specified as a `div` there is no support for the `label` and the component is read as `blank`, but when altered to `span` both NVDA and JAWS.

However, that said this fix does not provide context of the `label` when a user adds slotted text, so maybe a more robust fix is needed to override the slotted content?

For instance, in the example below with/without the fix `"Basemap"` is conveyed to assistive technologies, instead I'd expect the `label` or `"Gray basemap"` to be conveyed:

```html
<calcite-chip value="light-grey" label="Gray basemap">Basemap</calcite-chip>
```

Reply:
Found an example by Mozilla using `role="img"`, which seems very similar for our use case: https://codepen.io/svinkle/pen/oYjoNE.

As a result, we can:
- Move back to `div`
- Solve AT support for JAWS and NVDA
- When `label` is specified will override the slotted text

Updated commit coming shortly 👍🏻